www/src/components/Footer.astro

```diff
@@ -0,0 +1,137 @@
+---
+import Stack from '@/components/Stack/Stack.astro';
+import Text from '@/components/Text/Text.astro';
+import { SITE_TITLE } from '@/consts';
+import { menu } from '@/site.config';
+import SocialIcon from '@/components/SocialIcon.astro';
+import External from '@/components/External.astro';
+
+const year = new Date().getFullYear();
+const url = new URL(Astro.request.url);
+const isAbsolute = (path: string) => path.startsWith('https://');
+---
+
+<footer class="footer" aria-label="Main">
+  <div class="inner">
+    <Stack axis="y" gap={4}>
+      <Text variant="sm">
+        <span>© {year}</span>
+        <span>{SITE_TITLE}</span>
+      </Text>
+      <Text variant="xs" color="color-text-subdued">Deploys by Netlify</Text>
+    </Stack>
+
+    <div role="group">
+      <h5>Pages</h5>
+      <nav aria-label="Pages">
+        <ul class="ul">
+          {
+            menu.pages.map((link) => (
+              <li class="li">
+                <a
+                  aria-current={url.pathname === link.path ? 'page' : false}
+                  data-astro-prefetch
+                  class="link"
+                  href={link.path}
+                  aria-label={link.title}
+                  target={isAbsolute(link.path) ? '_blank' : undefined}
+                >
+                  <Fragment set:html={link.title} />
+                  {isAbsolute(link.path) ? <External /> : null}
+                </a>
+              </li>
+            ))
+          }
+        </ul>
+      </nav>
+    </div>
+
+    <div role="group">
+      <h5>Social</h5>
+      <nav class="social" aria-label="Social">
+        <ul class="ul">
+          {
+            menu.social.map((link) => (
+              <li class="li">
+                <a
+                  class="social-link"
+                  href={link.path}
+                  target={isAbsolute(link.path) ? '_blank' : undefined}
+                  title={link.title}
+                >
+                  <SocialIcon url={link.path} aria-hidden />
+                </a>
+              </li>
+            ))
+          }
+        </ul>
+      </nav>
+    </div>
+  </div>
+
+  <style lang="scss">
```

Comment:
@ChucKN0risK you can tell me to revert this, it’s fine. I just struggled already with a) making sure my classnames between components don’t have any duplicates/conflicts, and b) wanting to be lazy about naming without it bleeding into the global scope. The automatic scoping in each `.astro` component is just really good